Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -545,7 +548,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $this->listErrors($output, 'linting after fixing', $lintErrors);
         }
 
-        return !$resolver->isDryRun() || empty($changed) ? 0 : 3;
+        $exitStatus = 0;
+
+        if ($resolver->isDryRun()) {
+            if (!empty($invalidErrors)) {
+                $exitStatus |= self::EXIT_STATUS_FLAG_HAS_INVALID_FILES;
+            }
```

Comment:
hmm, is this the decision we made in #1211?
I'm still against putting this in `if isDryRun`. The question was:

> Question: do we want to exit with non-zero if invalid files were detected prior to fixing?

And most people voted with yes. Imo "prior to fixing" does not imply "but only in dry-run".